src/nxdoc/nuxeo-server/administration/database-configuration/documentdb.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Amazon DocumentDB
+labels:
+    - database
+    - documentdb
+tree_item_index: 700
+
+---
+The Nuxeo Platform supports Amazon DocumentDB
```

Comment:
This assertion should be check with product and support teams,
today DocumentDB claims to be compatible with MongoDB therefor it can be used by Nuxeo like a MongoDB backend but with few limitations.

Reply:
For now I removed it and stuck with the compatibility claim approach